Wrappers/Python/cil/optimisation/functions/Function.py

```diff
@@ -397,57 +454,70 @@ def L(self, value):
         super(SumScalarFunction, self.__class__).L.fset(self, value )
 
 class ConstantFunction(Function):
+
+    r"""  Function representing the Constant Function.
```

Comment:
There is an inconsistency in where the first word starts after the 3 `"` in all the docstrings. I believe it should be zero empty spaces.